src/test/java/com/epam/definitions/ReflectionDefinitions.java

```diff
@@ -22,90 +17,49 @@
-            e.printStackTrace();
-        }
+    public void intClass(final String className) {
 
+        clazz = getClazz(className);
+        object = getNewInstance(clazz);
     }
 
     @Then("Verify if current Object.getName equals {string}")
-    public void verifyClassName(String className) {
+    public void verifyClassName(final String className) {
 
         assertEquals(clazz.getName(), className);
     }
 
     @And("Set value - {int} in private field - {string}")
-    public void setValueInPrivateField(int fieldValue, String fieldName) {
-
-        try {
-            Field field = clazz.getDeclaredField(fieldName);
-            boolean flag = field.isAccessible();
-
-            field.setAccessible(true);
-            field.set(object, fieldValue);
-            field.setAccessible(flag);
-        } catch (NoSuchFieldException | IllegalAccessException e) {
-            e.printStackTrace();
-        }
+    public void setValueInPrivateField(final int fieldValue, final String fieldName) {
 
+        setIntPrivateField(object, fieldName, fieldValue);
     }
 
     @Then("Verify if method - {string} return value - {int}")
-    public void verifyReturnValueOfMethod(String methodName, int fieldValue) {
+    public void verifyReturnValueOfMethod(final String methodName, final int fieldValue) {
 
-        try {
-            assertEquals(clazz.getDeclaredMethod(methodName).invoke(object), fieldValue);
-        } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
-            e.printStackTrace();
-        }
+        assertEquals(invokeMethod(object, getDeclaredMethod(object, methodName)), fieldValue);
     }
 
     @Then("Verify if field - {string} return value - {int}")
-    public void verifyFieldValue(String fieldName, int value){
-
-        int declaredField = 0;
-
-        try {
-            Field field = clazz.getDeclaredField(fieldName);
-            boolean flag = field.isAccessible();
-            field.setAccessible(true);
-
-            declaredField = field.getInt(object);
-            field.setAccessible(flag);
-        } catch (NoSuchFieldException | IllegalAccessException e) {
-            e.printStackTrace();
-        }
-
-
-
-        assertEquals(declaredField, value);
+    public void verifyFieldValue(final String fieldName, final int value) {
 
+        assertEquals(getValueFromPrivateField(object, fieldName), value);
     }
 
-    @Then("Verify method = {string} with annotation field name - {string} and value false  and expected sum = {int}")
-    public void verifyAnnotationValue(String nameMethod, String fieldAnnName, int expectedSum) {
-
-        saveAnnotation(object);
+    @Then("Verify method = {string} with annotation field name - {string} and value false and expected sum = {int}")
+    public void verifyAnnotationValue(final String nameMethod, final String fieldAnnName, final int expectedSum) {
```

Comment:
We do not verify annotationValue here.

We want to verify methods are invoked depends on annotation value and final SUM is as expected
it should be name like

verifyTotalSumDependsOnInvocationState()